.github/stale.yml

```
@@ -0,0 +1,65 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - 🚀Preview Release
+  - 🚀Production Release
+  - 🚀QA Preview Release
+  - 🚀Hot Fix Release
+  - 🚀QA ProductionRelease
```

Fix this label as suggested below:

```suggestion
  - 🚀QA Production Release
```